python/tvm/relay/frontend/onnx.py

```diff
@@ -3444,6 +3444,35 @@ def body_fn(*loop_inputs):
         return outputs
 
 
+class LinearRegressor(OnnxOpConverter):
+    """Operator converter for LinearRegressor."""
+
+    @classmethod
+    def _impl_v1(cls, inputs, attr, params):
```

Comment:
document for parameter/return/function brief

Reply:
IMHO, given that all functions in this frontend do not have docstring, it should be fine for this PR to follow this convention for now.